### PR TITLE
Update renovate.json to group updates

### PR DESCRIPTION
**Pull request by acolchado**

### CODE REVIEW

File: renovate.json [hunk 1]
```diff
@@ -1,3 +1,34 @@
...
+    "packages:tslint"
+  ],
+  "packageRules": [
+    {
+      "packagePatterns": ["spur-"],
+      "groupName": "Spur packages",
+      "description": "All packages related to spur node framework"
+    },
+    {
+      "packagePatterns": ["prettier", "-prettier"],
```
Comment (terry-au) [inline renovate.json:19]: Would `prettier` also cover `prettier-`?

File: renovate.json [hunk 2]
```diff
@@ -1,3 +1,34 @@
...
+      "groupName": "Prettier packages",
+      "description": "All packages related to the prettier plugin"
+    },
+    {
+      "packagePatterns": ["eslint", "eslint-"],
+      "groupName": "ESLint packages",
+      "description": "All packages related to the eslint plugin"
+    },
+    {
+      "packagePatterns": ["jest", "@types/jest", "nock", "jest-"],
```
Comment (terry-au) [inline renovate.json:29]: Are you able to extend from Renovate's [built-in Jest + types group](https://docs.renovatebot.com/presets-group/#groupjestplustypes)? `group:jestPlusTypes`

**Review by terry-au — APPROVED** [review]
LGTM, just two comments but nothing blocking.

File: renovate.json (hunk 1 above)
Comment (acolchado) [reply renovate.json:19]: Unfortunately no. I'll have to recheck but it required it in another repo.

File: renovate.json (hunk 2 above)
Comment (acolchado) [reply renovate.json:29]: Hmm. I'll review this as a whole. I also recall Arnold mentioning that we have predefined config so I don't have to do this, but have not done the research yet. For now I am sticking to the common config we use.

**Pull request merged**